fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/ArithmeticMaker.java

```diff
@@ -58,32 +83,32 @@ public ArithmeticMaker(
      */
     public ArithmeticMaker(MetricDictionary metricDictionary, ArithmeticPostAggregationFunction function) {
         // TODO: Deprecate me, mappers should always be specified at creation time, not implicitly
```

Comment:
I don't know if I actually agree with this TODO.  Does anyone know why this is asked for?